src/vivarium/framework/results/manager.py

```diff
@@ -271,14 +266,22 @@ def register_binned_stratification(
             Keyword arguments for :meth: pandas.cut.
         """
 
-        def _bin_data(data: pd.Series | pd.DataFrame) -> pd.Series:
+        def _bin_data(data: pd.Series[int | float] | pd.DataFrame) -> pd.Series[Any]:
             """Use pandas.cut to bin continuous values"""
-            data = data.squeeze()
+            # mypy wants this squeeze to return a Series or DataFrame
+            if isinstance(data, pd.DataFrame):
+                data = data.squeeze()
             if not isinstance(data, pd.Series):
                 raise ValueError(f"Expected a Series, but got type {type(data)}.")
-            return pd.cut(
+            data = pd.cut(
                 data, bin_edges, labels=labels, right=False, include_lowest=True, **cut_kwargs
-            )
+            )  # type: ignore [call-overload]
+            if isinstance(data, pd.Series):
+                return data
+            else:
+                raise ValueError(
```

Comment:
I believe the only way this could happen would be if labels were False, but it's typed and described in the doctstring as always being a list of strings.

Reply:
If labels is False doesn't that mean the user implemented something incorrectly then?